contracts/shares/TurnupSharesV4.sol

```diff
@@ -150,6 +150,8 @@ contract TurnupSharesV4 is Initializable, OwnableUpgradeable {
 
   address[] internal _operators;
 
+  uint256 public constant WISH_EXPIRATION_TIME_V2 = 365 days;
```

Comment:
The storage looks fine. However, instead of adding a variable, I think that it is better to add a function that returns the expiration time, like
```
function _wishExpirationaTime() internal pure returns (uint256) {
  return 365 days;
}
```
so, in the future, that can be changed without altering the storage. Having a constant will save a bit of gas, but since the contract is on Polygon, the difference is negligible.

Reply:
Of course, if you add the function, leave the existing variable where it is. The contract will ignore it, but we need it to stay there to avoid issues with the storage schema.